python_modules/libraries/dagster-dbt/dagster_dbt/dbt_project.py

```diff
@@ -74,6 +74,74 @@ def prepare(self, project: "DbtProject") -> None:
 
 @experimental
 class DbtProject(DagsterModel):
+    """Representation of a dbt project and related settings that assist with managing manifest.json preparation.
+
+    By default, using this helps achieve a setup where
+        * during development, reload the manifest at run time to pick up any changes.
+        * when deployed, expect a manifest that was created at build time to reduce start-up time.
```

Comment:
The formatting of bullets looks kinda weird: https://al-04-12--dbt-dbtproject-apidoc.dagster.dagster-docs.io/_apidocs/libraries/dagster-dbt#dagster_dbt.DbtProject